nibabel/openers.py

```diff
@@ -18,16 +18,28 @@
 
 # is indexed_gzip present and modern?
 try:
-    from indexed_gzip import SafeIndexedGzipFile, __version__ as version
+    import indexed_gzip as igzip
+    version = igzip.__version__
 
     HAVE_INDEXED_GZIP = True
 
+    # < 0.6 - no good
     if StrictVersion(version) < StrictVersion('0.6.0'):
         warnings.warn('indexed_gzip is present, but too old '
                       '(>= 0.6.0 required): {})'.format(version))
         HAVE_INDEXED_GZIP = False
-
-    del version
+    # < 0.7 - does not support drop_handles
+    elif StrictVersion(version) < StrictVersion('0.7.0'):
+        class IndexedGzipFile(igzip.SafeIndexedGzipFile):
+            def __init__(self, *args, **kwargs):
+                kwargs.pop('drop_handles', None)
+                super(IndexedGzipFile, self).__init__(*args, **kwargs)
```

Comment:
I'm uncomfortable with this, as we're now making the default non-closing `IndexedGzipFile`s. The reason we previously had `KEEP_FILE_OPEN_DEFAULT = False` was because users could unexpectedly end up with a bunch of open filehandles by installing `indexed_gzip`. So that will be the case if users have `indexed_gzip.__version__ < 0.7.0`.

I think our only options are to either make 0.7 the minimum version (in line 27), or to make the value of `KEEP_FILE_OPEN_DEFAULT` depend on the version of `indexed_gzip`.

Reply:
So ... given neurodebian I guess the latter option is the way t o go ...

Reply:
Well, we can always just do the same as for < 0.6, and warn. It's not that people using Debian can't upgrade packages with `pip`.

Reply:
If you're happy with that, I am!

Reply:
That would be my preference, just because it's simpler, and makes constants a little more constant.

But we should also go ahead and argue for NeuroDebian updating its package.